src/app/App.js

```diff
@@ -0,0 +1,173 @@
+define([
+    'agrc/widgets/locate/FindAddress',
+    'agrc/widgets/locate/MagicZoom',
```

Comment:
`FindAddress` & `MagicZoom` are unused and should be removed from this file as well as `App.styl`.